stripper/ze_bathroom_v2_5.cfg

```diff
@@ -0,0 +1,71 @@
+;Timer Fix
+
+modify:
+{
+	match:
+	{	
+	"targetname" "skrin_trigger_1"
+	"classname" "trigger_once"
+	"hammerid" "496571"
+	}
+	delete:
+	{
+	"OnStartTouch" "consolaCommandsay Resist for 25 seconds.01"
+	}
+	insert:
+	{
+	"OnStartTouch" "consolaCommandsay Resist for 30 seconds.01"
+	}
+}
+
+modify:
+{
+	match:
+	{	
+	"classname" "trigger_once"
+	"hammerid" "496601"
+	}
+	delete:
+	{
+	"OnStartTouch" "consolaCommandsay Resist for 20 seconds.01"
+	}
+	insert:
+	{
+	"OnStartTouch" "consolaCommandsay Resist for 25 seconds.01"
+	}
+}
+
+modify:
+{
+	match:
+	{	
+	"classname" "trigger_once"
+	"hammerid" "567231"
+	}
+	delete:
+	{
+	"OnStartTouch" "consolaCommandsay Resist for 25 seconds.01"
+	}
+	insert:
+	{
+	"OnStartTouch" "consolaCommandsay Resist for 30 seconds.01"
+	}
+}
+
+modify:
+{
+	match:
+	{	
+	"targetname" "lvl1_koskapada"
+	"classname" "trigger_once"
+	"hammerid" "497300"
+	}
+	delete:
+	{
+	"OnStartTouch" "consolaCommandsay Hold here until something happens!01"
+	}
+	insert:
+	{
+	"OnStartTouch" "consolaCommandsay Hold here until the blue sponge drops, then get in the tub!01"
+	}
+}
```

Comment:
We can bring this in, however I'm going to merge it with our current stripper after the PR is accepted.